exc:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers ---------------------------------------
  wheel_fr_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_fr_joint
    pid: {p: 10000.0, i: 500.0, d: 0.0}

  wheel_fl_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_fl_joint
    pid: {p: 10000.0, i: 500.0, d: 0.0}

  wheel_rr_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_rr_joint
    pid: {p: 10000.0, i: 500.0, d: 0.0}

  wheel_rl_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_rl_joint
    pid: {p: 10000.0, i: 500.0, d: 0.0}

  turntable_position_controller:
    type: effort_controllers/JointPositionController
    joint: turn_to_base
    pid: {p: 10000.0, i: 0.0, d: 1000.0}

  lower_arm_position_controller:
    type: effort_controllers/JointPositionController
    joint: lower_to_turn
    pid: {p: 10000.0, i: 0.0, d: 1000.0}

  upper_arm_position_controller:
    type: effort_controllers/JointPositionController
    joint: upper_to_lower
    pid: {p: 10000.0, i: 0.0, d: 1000.0}
